src/nifreeze/model/gpr.py

```diff
@@ -332,7 +334,7 @@ def __call__(
 
         return self.beta_l * C_theta, K_gradient
 
-    def diag(self, X: np.ndarray) -> np.ndarray:
```

Comment:
?

Reply:
https://github.com/nipreps/nifreeze/pull/28/commits/8341c3087ffb1709f202ad274452dcd5c81cb9e4

Reply:
Fixed in bd38772.

Reply:
Thanks @effigies !